src/com/generic/page/Cart.java

```
@@ -200,22 +201,45 @@ public static void editOptions() throws Exception {
 			// Editing the product
 			try {
 				// Check if the product has drop-down and edit it
-				SelectorUtil.initializeSelectorsAndDoActions(CartSelectors.optionsDropDown.get(), "FFF2");
+				if (isMobile()) {
+					SelectorUtil.initializeSelectorsAndDoActions(CartSelectors.optionsDropDown.get(), "FFF2");
+				} else {
+					List<WebElement> selects = SelectorUtil.getElementsList(CartSelectors.optionsDropDown.get());
+					for (int i = 0; i < selects.size(); i++) {
+						WebElement ele = selects.get(i);
+						Select fruits = new Select(ele);
+						fruits.selectByIndex(2);
+					}
+				}
 
 			} catch (Exception e) {
 
 				try {
-
 					// Check if the product has swatches and select one
-					SelectorUtil.initializeSelectorsAndDoActions(CartSelectors.optionsImage.get(), "index,1");
+					// SelectorUtil.initializeSelectorsAndDoActions(CartSelectors.optionsImage.get(),
+					// "index,1");
+					List<WebElement> imgs = getDriver().findElements(By.cssSelector(CartSelectors.optionsImage.get()));
+					int x;
+					if (imgs.size() < 3)
+						x = 1 / 0;

```

Please throw a new exception !